packages/graphile-build-pg/src/plugins/PgTypesPlugin.js

```diff
@@ -461,6 +511,16 @@ export default (function PgTypesPlugin(
         650: CidrType,
         829: MacAddrType,
         774: MacAddr8Type,
+        24: RegProcType,
+        2202: RegProcedureType,
+        2203: RegOperType,
+        2204: RegOperatorType,
+        2205: RegClassType,
+        2206: RegTypeType,
+        4096: RegRoleType,
+        4089: RegNamespaceType,
+        3734: RegConfigType,
+        3769: RegDictionaryType,
```

Comment:
Normally I'd gate this with a feature flag so existing schemas aren't impacted by these new types, but I haven't heard of anyone using these reg values before within a PostGraphile schema and this PR fixes a bug in their handling which implies it won't previously have been working (I think?), so from what I understand this won't break existing schemas because no-one can successfully use these types already. Is that accurate?

Reply:
Well, this PR fixes a very specific bug in handling these reg values when part of a primary key. We've previously used these types before (not being part of a primary key) but they would incorrectly be typed as `BigFloat` which is also a string like type. So, I believe that it will result in a different schema, but this would be semantics.

Reply:
It'd break operations where the type is declared amongst the variables (`query MyQuery ($regConfig: BigFloat) { ... }`), but IMO if you're doing that you can clearly see it's wrong and should have filed an issue long ago ;)

We can always add a retro-active flag if it affects anyone.